trolley/drawers.lua

```diff
@@ -0,0 +1,64 @@
+local INT, STRING, FLOAT, TABLE  =
+	drawers.trolley_META_TYPE_INT,
+	drawers.trolley_META_TYPE_STRING,
+	drawers.trolley_META_TYPE_FLOAT,
+	drawers.trolley_META_TYPE_TABLE
+
+local to_register = { }
+
+if core.get_modpath("default") and default then
+	to_register = {"drawers:wood",
+    "drawers:acacia_wood",
```

Comment:
this one is indented different from all the rest

Reply:
done